app/Models/Course.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\SoftDeletes;
+
+class Course extends Model
+{
+    use HasFactory, SoftDeletes;
+
+    protected $table = 'courses';
+
+    protected $fillable = [
+        'title',
+        'logo-path',
+        'description',
+        'intro',
+        'learn_time',
+        'quizzes',
+        'price',
+        'deleted_at',
+    ];
+
+    public function lesson()
```

Comment:
đặt lại tên nhé $this->hasMany(Lessons::class); trả về nhiều lesson nên function name lessons phải có `s` tương tự sửa các phần dưới nhé